xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1624,10 +1743,10 @@ The comparison of the JSON token value in the source and the lookup text is done
         <Parameter Name="text" Type="System.ReadOnlySpan&lt;System.Char&gt;" />
       </Parameters>
       <Docs>
-        <param name="text">The UTF-8 encoded text to compare against.</param>
-        <summary>Compares the UTF-8 encoded text in a read-only character span to the unescaped JSON token value in the source and returns a value that indicates whether they match.</summary>
+        <param name="text">The text to compare against.</param>
```

Comment:
This was incorrect. `ROS<char>` doesn't contain UTF-8 text, but rather UTF-16 text (like the string overload).